nuitka/distutils/bdist_nuitka.py

```diff
@@ -53,7 +53,15 @@ def run(self):
         # for .py_modules
 
         self.compile_packages = self.distribution.packages
-        self.main_package = self.compile_packages[0]
+        if self.compile_packages:
+            self.main_target = self.compile_packages[0]
+            self.py_modules = False
+        else:
```

Comment:
Like I said, I am not convinced that not both are allowed to be used at the same time. Can you check what happens if one specifies both?

Reply:
I've included a new distutils example `package_and_module`. Nuitka builds it without error but `ls package_and_module/build/lib/` outputs 

```
example1_package/  libsome_module.a  some_module.pyd*  some_module.pyi
```

Which seems to be missing the .pyd and .pyi files from the package.